### PR TITLE
Some misspellings corrected through the part 2 document

### PR DESCRIPTION
**Pull request by alx741**

### CODE REVIEW

**Comment by Huddayn** [comment]
```
 - Memory segmentation was completely redone in the protected mode.
 + Memory segmentation was completely remade in the protected mode.
```

I find 'redone' more fitting here.

**Comment by alx741** [comment]
Thank you @Huddayn , changing...

**Comment by alx741** [comment]
So 'its', 'is', and 'an' are the changes, someone can confirm if its better?

**Comment by 0xAX** [comment]
@alx741 @Haddayn Thank you.

@alx741 I like this changes, can you please rebase on master and i will merge it.

File: linux-bootstrap-2.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Don't worry, i know that it looks little scary after real mode, but it's easy. L
 | 15          1    1    1   1 | Code            | Execute/Read, conforming, accessed
 ```
 
-As we can see first bit is 0 for data segment and 1 for code segment. Next three bits `EWA` are expansion direction (expand-down segment will grow down, more about it you can read [here](http://www.sudleyplace.com/dpmione/expanddown.html)), write enable and accessed for data segments. `CRA` bits are conforming (A transfer of execution into a more-privileged conforming segment allows execution to continue at the current privilege level), read enable and accessed.
+As we can see first bit is 0 for data segment and 1 for code segment. Next three bits `EWA` are expansion direction (expand-dow segment will grow down, you can read more about this [here](http://www.sudleyplace.com/dpmione/expanddown.html)), write enable and accessed for data segments. `CRA` bits are conforming (A transfer of execution into a more-privileged conforming segment allows execution to continue at the current privilege level), read enable and accessed.
```
Comment (0xAX) [inline linux-bootstrap-2.md:95]: There is must be expand-down

**Comment by alx741** [comment]
hmm... I've actually introduced new misspellings... closing and re-trying latter with new stuff

**Pull request closed without merging**